### DISCUSSION TITLE
Using the width tag in a rendering

### DISCUSSION THREAD

**Discussion by ravenfeld** [osmandapp/OsmAnd#12639] [Q&A]
Hello, I'm trying to use the width tag in a render to differentiate the size of the paths but I can't. Does anyone have an idea?
I would like to differentiate the paths of less than 1m, 1m and 2m and more than 2m width.

Thanks in advance.

**Comment by vshcherb** [comment]
It's not easily possible.
- First of all maps need to be prepared with additional tag like 'width=1', 'width=2', 'width=3' - you can only enumerate certain values with rendering_types.xml
- Secondly you can create switch in render.xml and specify different pixel width for these attributes
